config/cmaq/NEXUS_Config.rc

```diff
@@ -1869,337 +1865,337 @@ Cap time shift              : true
-0 NEI2016v1_PT_C12MARINE_PCL            $ROOT/NEI2016v1/v2020-07/$MM/NEI2016v1_0.1x0.1_2016$MM$DD_cmv_c1c2_12.nc          PCL           2016/1-12/1-31/$HH C xyz  kg/m2/s  PCL         1007     1 50
-0 NEI2016v1_PT_C12MARINE_PFE            $ROOT/NEI2016v1/v2020-07/$MM/NEI2016v1_0.1x0.1_2016$MM$DD_cmv_c1c2_12.nc          PFE           2016/1-12/1-31/$HH C xyz  kg/m2/s  PFE         1007     1 50
-0 NEI2016v1_PT_C12MARINE_PH2O           $ROOT/NEI2016v1/v2020-07/$MM/NEI2016v1_0.1x0.1_2016$MM$DD_cmv_c1c2_12.nc          PH2O          2016/1-12/1-31/$HH C xyz  kg/m2/s  PH2O        1007     1 50
-0 NEI2016v1_PT_C12MARINE_PK             $ROOT/NEI2016v1/v2020-07/$MM/NEI2016v1_0.1x0.1_2016$MM$DD_cmv_c1c2_12.nc          PK            2016/1-12/1-31/$HH C xyz  kg/m2/s  PK          1007     1 50
-0 NEI2016v1_PT_C12MARINE_PMG            $ROOT/NEI2016v1/v2020-07/$MM/NEI2016v1_0.1x0.1_2016$MM$DD_cmv_c1c2_12.nc          PMG           2016/1-12/1-31/$HH C xyz  kg/m2/s  PMG         1007     1 50
-0 NEI2016v1_PT_C12MARINE_PMN            $ROOT/NEI2016v1/v2020-07/$MM/NEI2016v1_0.1x0.1_2016$MM$DD_cmv_c1c2_12.nc          PMN           2016/1-12/1-31/$HH C xyz  kg/m2/s  PMN         1007     1 50
-0 NEI2016v1_PT_C12MARINE_PMOTHR         $ROOT/NEI2016v1/v2020-07/$MM/NEI2016v1_0.1x0.1_2016$MM$DD_cmv_c1c2_12.nc          PMOTHR        2016/1-12/1-31/$HH C xyz  kg/m2/s  PMOTHR      1007     1 50
-0 NEI2016v1_PT_C12MARINE_PNA            $ROOT/NEI2016v1/v2020-07/$MM/NEI2016v1_0.1x0.1_2016$MM$DD_cmv_c1c2_12.nc          PNA           2016/1-12/1-31/$HH C xyz  kg/m2/s  PNA         1007     1 50
-0 NEI2016v1_PT_C12MARINE_PNCOM          $ROOT/NEI2016v1/v2020-07/$MM/NEI2016v1_0.1x0.1_2016$MM$DD_cmv_c1c2_12.nc          PNCOM         2016/1-12/1-31/$HH C xyz  kg/m2/s  PNCOM       1007     1 50
-0 NEI2016v1_PT_C12MARINE_PNH4           $ROOT/NEI2016v1/v2020-07/$MM/NEI2016v1_0.1x0.1_2016$MM$DD_cmv_c1c2_12.nc          PNH4          2016/1-12/1-31/$HH C xyz  kg/m2/s  PNH4        1007     1 50
-0 NEI2016v1_PT_C12MARINE_PNO3           $ROOT/NEI2016v1/v2020-07/$MM/NEI2016v1_0.1x0.1_2016$MM$DD_cmv_c1c2_12.nc          PNO3          2016/1-12/1-31/$HH C xyz  kg/m2/s  PNO3        1007     1 50
-0 NEI2016v1_PT_C12MARINE_PTI            $ROOT/NEI2016v1/v2020-07/$MM/NEI2016v1_0.1x0.1_2016$MM$DD_cmv_c1c2_12.nc          PTI           2016/1-12/1-31/$HH C xyz  kg/m2/s  PTI          1007    1 50
-0 NEI2016v1_PT_C12MARINE_PSI            $ROOT/NEI2016v1/v2020-07/$MM/NEI2016v1_0.1x0.1_2016$MM$DD_cmv_c1c2_12.nc          PSI           2016/1-12/1-31/$HH C xyz  kg/m2/s  PSI          1007    1 50
-0 NEI2016v1_PT_C12MARINE_PMC            $ROOT/NEI2016v1/v2020-07/$MM/NEI2016v1_0.1x0.1_2016$MM$DD_cmv_c1c2_12.nc          PMC           2016/1-12/1-31/$HH C xyz  kg/m2/s  PMC          1007    1 50
-0 NEI2016v1_PT_C12MARINE_PSO4           $ROOT/NEI2016v1/v2020-07/$MM/NEI2016v1_0.1x0.1_2016$MM$DD_cmv_c1c2_12.nc          PSO4          2016/1-12/1-31/$HH C xyz  kg/m2/s  PSO4         1007    1 50
```

Comment:
Shouldn't the old point sources we are still using be 'xyz' and not 'xy'?

Reply:
If you remember the original point source files here actually had 3 layers to them.  We had collapsed them into a single value and had just an xy dimension.  Wtih xyz it fails